packages/frontend/src/hooks/useProject.ts

```diff
@@ -58,20 +64,19 @@ export const useUpdateMutation = (id: string) => {
 };
 
 export const useCreateMutation = () => {
-    const queryClient = useQueryClient();
-    const { showToastSuccess } = useApp();
-    return useMutation<Project, ApiError, UpdateProject>(
+    const { setActiveJobId, showToastError } = useApp();
+    return useMutation<ApiJobStartedResults, ApiError, UpdateProject>(
         (data) => createProject(data),
         {
             mutationKey: ['project_create'],
-            onSuccess: async () => {
-                await queryClient.invalidateQueries(['projects']);
-                await queryClient.invalidateQueries([
-                    'projects',
-                    'defaultProject',
-                ]);
-                showToastSuccess({
-                    title: `Success! New project was created`,
+            retry: 3,
```

Comment:
why do we need retry here ?

Reply:
If the network is unstable, I thought it would be good to try a few times to check job status before returning an error, since if we can't poll the job once or twice, it won't be critical